docker-wrappers/DOMINO/README.md

```diff
@@ -0,0 +1,16 @@
+# DOMINO Docker image
+
+A Docker image for [DOMINO](https://github.com/Shamir-Lab/DOMINO) that is available on [DockerHub](https://hub.docker.com/repository/docker/otjohnson/domino).
+
```

Comment:
This readme is one place where we can briefly document some implementation decisions, like merging all output modules into a single pathway.